wiki/configuration/advanced-configuring/en.md

```diff
@@ -129,15 +148,123 @@ Here we leverage a `ValueMap` instead of a `String` as the serialized type. This
+
+`src/main/com/example/example_mod/ExampleModConfig`:
+
+```java
+@Processor("processConfig")
+public class ExampleModConfig extends ReflectiveConfig {
+    // ...
+    public static class AdvancedSettings extends Section {
+        // ...
+        @Processor("processPrintStream")
+        public final TrackedValue<PrintStreamOption> printStream = this.value(PrintStreamOption.SYSTEM_OUT);
+        public transient PrintStream activeStream = printStream.value().getStream();
+
+        public void processPrintStream(TrackedValue.Builder<PrintStreamOption> builder) {
+            builder.callback(value -> activeStream = printStream.value().getStream());
+        }
+		
+        // ...
+    }
+}
+```
 
-g
+Using our callback, we update the `activeStream` variable each time that the print stream is changed. This keeps it perfectly in sync with the `printStream` field at all times! Note that we mark it as `transient`, a keyword which tells Java (and subsequently Quilt Config!) not to serialize the value.
+Now instead of dealing with `ExampleModConfig.INSTANCE.advancedSettings.printStream.value().getStream()` we can simply do `ExampleModConfig.INSTANCE.advancedSettings.activeStream`, simplifying our lives a little when interacting with the config. The power of processors, in action.
 
 ## Changing the config format
 
-Let's get into how you choose a file format to save to. Quilt Config currently only provides two serializers: `json5`, an extension of the JSON format to allow cleaner syntax and comments, and `toml`, which is the default format it serializes to. If you want to switch to `json5`, we can do that using a `Processor`!
+Let's get into how you choose a file format to save to. Quilt Config currently only provides two serializers: `json5`, an extension of the JSON format to allow cleaner syntax and comments, and `toml`, with the default being `toml`. If you want to switch to `json5`, we can do that using a `Processor`!
+We'll need to apply this processor globally to our config, since the way we'll be changing the format is via the `Config.Builder` object a config class processor will provide.
 
 This processor needs to run before the config is read, so we're going to place it directly on the class:
 
-https://github.com/hibiii/BlindMe/blob/main/src/main/java/hibi/blind_me/Config.java use processor to run builder.format
+`src/main/com/example/example_mod/ExampleModConfig`:
+
+```java
+@Processor("processConfig")
+public class ExampleModConfig extends ReflectiveConfig {
+	public void processConfig(Config.Builder builder) {
+		// ...
+        builder.format("json5");
+	}
+
+	// ...
+}
+```
+
+With our knowledge of processors, this is simple! You can also use the config builder to add new fields, new sections, and update metadata (TODO LINK TO METADATA PAGE), on top of changing the format and using callbacks as we've already covered.
```

Comment:
Still a todo here

Reply:
well yeah because I haven't written that other page yet

Reply:
ah i thought the PR was done. makes sense.